### ISSUE TITLE
Menu functionality appears to be undocumented for the user

### ISSUE THREAD

**Issue by bmintz** [mikeryan/ems-flasher#7]
How can I use the menu functionality? There's no --help info, and it's not in the manpage. Is it even complete?
Can I just do this:
```sh
cat menu.gb i-can-assure-you-i-ripped-this.gb not-pirated-and-totally-legit-game2.gbc > no-copyright-infringement-here-menu.gbc
```
and then flash `no-copyright-infringement-here-menu.gbc`? If so, to which bank?

If the project needs user-facing documentation, I'm happy to write it, as long as it doesn't involve interpreting code lol. I don't do C normally.

**Comment by bmintz** [comment]
Hold on. I think I understand. I just have to pass multiple filenames as the ROM argument. OK.

Please leave this issue open. I'll use it as an attempt to document the features.

**Comment by mikeryan** [comment]
closed by 039d321, damn typo
